vision/util/blob_logger.py

```diff
@@ -0,0 +1,36 @@
+"""Utility for logging blobs detected in an image"""
+
+import cv2
+import numpy as np
+try:
+    from vision.blob.blobfind import BlobFinder
+except ImportError:
+    from blob.blobfind import BlobFinder
+
+def log_blobs(blob_finder):
+    """
+    Shows the detected blobs in an image and print the bounding boxes
+
+    Parameters
+    ----------
+    blob_finder: BlobFinder
+        BlobFinder object that contains an image and potentially params
```

Comment:
Lets change this to a list of rectangles, which the BlobFinder outputs. This will make it generic to anyone's algorithm. In the if __name__ == '__main__' of blob you could run blob finder and have it print this.

May need to pass in an image too.